test/scripts/status.test.js

```diff
@@ -23,22 +24,20 @@ contract('status command', function([_, owner]) {
   const contractsData = [{ name: contractName, alias: contractAlias }]
   const anotherContractName = "AnotherImplV1";
   const stdlibNameVersion = 'mock-stdlib@1.1.0';
-  const packageFileName = "test/tmp/zos.json";
-  const networkFileName = `test/tmp/zos.${network}.json`;
-  
-  beforeEach('cleanup', async function() {
-    cleanup(packageFileName)
-    cleanup(networkFileName)
+  const tmpDir = "test/tmp";
+  const packageFileName = `${tmpDir}/zos.json`;
+  const networkFileName = `${tmpDir}/zos.${network}.json`;
+
+  beforeEach('setup', async function() {
+    fs.createDir(tmpDir);
```

Comment:
if we need to do this in all tests we can move it to the `setup.js` file, do you agree?

Reply:
Yes, I would love to make it common. But I'm not finding any docs about setup.js. How can we move a `beforeEach` and `afterEach` there? And can they return a variable for the tests to use?

Reply:
we can take care of it later then :/